pymeasure/instruments/agilent/agilentE4407B.py

```diff
@@ -0,0 +1,637 @@
+        ":DISP:ANN:CLOCK:DATE:FORMAT %g",
+        """ A property that represents the date format of the instrument.""",
+        validator=strict_discrete_set,
+        values=["MDY", "DMY"],
+    )
+
+    display_time = Instrument.control(
+        ":DISP:ANN:CLOCK?",
+        ":DISP:ANN:CLOCK %g",
+        """Turn off or on the time display on the instrument.""",
+        validator=strict_discrete_set,
+        values=["ON", "OFF", 0, 1],
+    )
+
+    display_title = Instrument.control(
+        ":DISP:ANN:TITLE:DATA?",
+        ":DISP:ANN:TITLE:DATA %g",
+        """Set the text of the title display on the instrument.""",
+    )
+
+    display = Instrument.setting(
+        ":DISP:ENABLE %g",
+        """Turn off or on the display on the instrument.""",
+        validator=strict_discrete_set,
+        values=["ON", "OFF", 0, 1],
+    )
+    # Mesurement commands
+    # Fetch commands
+    # Mass memory commands
+    catalog = Instrument.measurement(
+        ":MMEM:CAT? %g",
+        """A command that returns the list of mass memory files in specified drive.""",
+        validator=strict_discrete_set,
+        values=["A:", "C:"],
+    )
+
+    def copy_file(self, source, destination):
+        """
+        A command that copies a file to another.
+        """
+        self.write(":MMEM:COPY %s,%s" % (source, destination))
+
+    def send_file(self, filename, data_block):
+        """
+        A command that sends a file to the instrument.
+        """
+        self.write(f":MMEM:DATA {filename},{data_block}")
+
+    recive_file = Instrument.measurement(
+        ":MMEM:DATA? %g",
+        """A command that returns the contents of a file.""",
+    )
+    delet_file = Instrument.setting(
+        ":MMEM:DEL %g",
+        """A command that deletes a file from the mass memory.""",
+    )
+
+    save_screen = Instrument.setting(
+        ":MMEM:STOR:SCR %g",
+        """Save the current screen to the specified mass memory. eg C:myscreen.gif""",
+    )
+
+    def get_screen(self):
+        """
+        A command that returns the contents of the screen.
+        """
+        self.save_screen("C:tempScreen.gif")
+        data = self.recive_file("C:tempScreen.gif")
+        self.delet_file("C:tempScreen.gif")
+        return data
+
+    # Format commands
+    byte_order = Instrument.control(
+        ":FORM:BORD?",
+        ":FORM:BORD %g",
+        """ A property that controls the binary data byte order for transfer.""",
+        validator=strict_discrete_set,
+        values=["NORM", "SWAP"],
+    )
+
+    num_format = Instrument.control(
+        ":FORM:DATA?",
+        ":FORM:DATA %g",
+        """ A property that controls the numerical data format for transfer.""",
+        validator=strict_discrete_set,
+        values=["ASC", "ASCII", "INT,32", "REAL,32", "REAL,64"],
+    )
+    # hard copy commands
+    # initaite commands
+    # input commands
+    # unit commands
+    mesure_units = Instrument.control(
```

Comment:
```suggestion
    measure_units = Instrument.control(
```
but maybe better "measurement_units"?